### PR TITLE
#56 preferences styling

### PR DESCRIPTION
**Pull request by PrashanthKumarM**
Fixed build as well

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,8 +11,9 @@ android:
     - extra-google-m2repository
     - addon-google_apis-google-16
 before_script:
-  - cd actionlist && npm install
+  - cd actionlist && rm -rf node_modules && npm install
```
Comment (burnumd) [inline .travis.yml]: Why are you having to do these steps in CI? Is it related to [this](https://github.com/petester42/jest-preset-typescript-react-native/issues/1)? If so, updating to the latest `react-native` may help. I'd hate to see workarounds for something that's fixed (and if that doesn't fix it, it needs a comment about why this workaround is present).

File: actionlist/app/reducers/action_items.js [hunk 2]
```diff
@@ -8,15 +8,15 @@ export const defaultState = {
   drawerExpanded: false,
   optionSelected: '',
   dropdownColors: {
-    saved: Colors.White,
```
Comment (burnumd) [inline actionlist/app/reducers/action_items.js:11]: What's the purpose of these changes? Having capitalized object keys seems pretty non-standard.

**Review by burnumd — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (PrashanthKumarM) [reply .travis.yml]: Hi @burnumd it seems to be a problem with react-native-router-flux. It has its own react-native in its node_modules. While the line highlighted here is not needed, the line after this is needed. The @providesModule takes the reference of both the react-natives and says there is a collision. Deleting the node_module of react native router flux solves the problem

Comment (PrashanthKumarM) [reply .travis.yml]: Both the pull requests have the same problem with the build and I guess all the further builds will have it too. I am not able to figure out the root cause to this.

File: actionlist/app/reducers/action_items.js (hunk 2 above)
Comment (PrashanthKumarM) [reply actionlist/app/reducers/action_items.js:11]: changes made

**Comment by PrashanthKumarM** [comment]
@burnumd The code climate issues are not valid. They are propType validations for 2 different elements

File: .travis.yml (hunk 1 above)
Comment (PrashanthKumarM) [reply .travis.yml]: @burnumd is it not ok to just delete the node modules folder? Our changes had no effect in the build failing.

Comment (PrashanthKumarM) [reply .travis.yml]: correction: node_modules folder inside react-native-router-flux

Comment (burnumd) [reply .travis.yml]: In that case, there should be a comment explaining why file system operations are occurring, hopefully with a link to an outstanding issue in the offending dependency. In this case: https://github.com/facebook/jest/issues/2070

File: actionlist/app/reducers/action_items.js [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
-    Saved: Colors.White,
-    Initiated: Colors.White,
-    Disapproved: Colors.White,
-    Enroute: Colors.White,
-    Approved: Colors.White,
-    Final: Colors.White,
-    Processed: Colors.White,
-    Exception: Colors.White,
-    Cancel: Colors.White,
+    Saved: 'white',
```
Comment (burnumd) [inline actionlist/app/reducers/action_items.js]: Wait, why does this no longer use `Colors.white`?

File: actionlist/app/reducers/action_items.js [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
-    saved: Colors.White,
-    initiated: Colors.White,
-    disapproved: Colors.White,
-    enroute: Colors.White,
-    approved: Colors.White,
-    final: Colors.White,
-    processed: Colors.White,
-    exception: Colors.White,
-    cancel: Colors.White,
+    Saved: 'white',
```
Comment (burnumd) [inline actionlist/app/reducers/action_items.js]: Why do these no longer use `Colors.white`?

File: actionlist/package.json [hunk 5]
```diff
@@ -16,14 +16,9 @@
...
     "@fugood/react-native-call-modal": "0.0.6",
-    "camel-case": "^3.0.0",
-    "change-case": "^3.0.1",
     "react": "15.4.2",
-    "react-addons-test-utils": "^15.4.2",
     "react-native": "0.40.0",
-    "react-native-button": "^1.8.2",
-    "react-native-collapsible": "^0.8.0",
-    "react-native-drawer": "^2.3.0"
+    "react-native-button": "^1.8.2"
```
Comment (burnumd) [inline actionlist/package.json]: Does this dependency need to be a production dependency?

**Review by burnumd — CHANGES_REQUESTED** [review]

File: actionlist/package.json (hunk 5 above)
Comment (PrashanthKumarM) [reply actionlist/package.json]: moved to dev dependency

File: actionlist/app/reducers/action_items.js (hunk 4 above)
Comment (PrashanthKumarM) [reply actionlist/app/reducers/action_items.js]: changed to Colors.white

**Review by burnumd — APPROVED** [review]

**Pull request merged**